README.md

```diff
@@ -6,17 +6,7 @@ Welcome to Extension Workshop, a launchpad for building Firefox extensions! 🚀
 
 ## Updating Content
```

Comment:
I think we can change it to #Contributing. WDYT @caitmuenster

Reply:
I think this is ok as is. :)